src/golang/lib/job/spec.go

```diff
@@ -174,6 +175,15 @@ type DiscoverSpec struct {
 	OutputContentPath string              `json:"output_content_path"  yaml:"output_content_path"`
 }
 
+type CompileAirflowSpec struct {
+	BasePythonSpec
+	OutputContentPath string              `json:"output_content_path"  yaml:"output_content_path"`
+	DagId             string              `json:"dag_id"  yaml:"dag_id"`
```

Comment:
what is `dag id` for?

Reply:
It's an Airflow concept. It's basically the name of the workflow, but for Airflow.